### PR TITLE
Documentation updates

### PR DESCRIPTION
**Pull request by harimohanr**
Documentation updates

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -3,4 +3,33 @@ layout: home
...
 ---
 
-# BeSecure Schema
+### Be-Secure Schema Definitions
+
+An easy-to-use data format that maps precisely to various data exchange needs between cybersecurity analysts and security labs. These schema definitions address many of the shortcomings of dealing with vulnerabilities in open source software and ML models in Be-Secure ecosystem.
+
+These unified format means that vulnerability databases, open source users, and security researchers can easily share tooling and consume vulnerabilities across all of open source. This means a more complete view of vulnerabilities in open source for everyone, as well as faster detection and remediation times resulting from easier automation.
+
+Key Be-Secure assets these schema definitions cater to are as below.
```
Comment (panickervinod) [inline docs/index.md:12]: reword BeS Assets as BeS Eco System Entities

File: docs/index.md [hunk 2]
```diff
@@ -3,4 +3,33 @@ layout: home
...
+- Open source projects of interest(OSSPoI)
+- Open source vulnerabilities of interest( OSSVoI)
+- Open source models of Interest (OSMoI)
+
+Key tooling Be-Secure offers..
```
Comment (panickervinod) [inline docs/index.md:17]: Remove Key tooling , All are BeS Entities

File: docs/index.md [hunk 3]
```diff
@@ -3,4 +3,33 @@ layout: home
...
+- BeS Environments
+- BeS Playbooks
+- BeS Assessments
+- BeS Exploit & Attack Simuation (EAS)
+- BeSLabs
+
+# Schema Definitions
```
Comment (panickervinod) [inline docs/index.md:24]: Schema Defintion for BeS Eco System Entities.

**Review by panickervinod — COMMENTED** [review]
Reword BeS Assest To BeS Eco Sytem Entities

File: docs/index.md (hunk 1 above)
Comment (harimohanr) [reply docs/index.md:12]: Change made as per the feedback

File: docs/index.md (hunk 2 above)
Comment (harimohanr) [reply docs/index.md:17]: Change made as per the feedback

**Pull request merged**